### ISSUE TITLE
[Documentation] Setting up local dev environment failing to run `yarn build`

### ISSUE THREAD

**Issue by nmummau** [vuetifyjs/vuetify#11169]
### Environment

**Browsers:** Chrome 81.0.4044.113
**OS:** Linux x86_64

### Steps to reproduce
Install node v13.13.0
Install yarn v1.22.4 with instructions here - https://classic.yarnpkg.com/en/docs/install/#debian-stable
Fork Vuetify git repo.
Follow Vuetify docs for setting up a local dev environment.
`yarn`, success:
```
$ yarn
yarn install v1.22.4
[1/5] Validating package.json...
[2/5] Resolving packages...
success Already up-to-date.
$ node scripts/post-install.js
Done in 0.66s.
```
`yarn build`, get an error:
```
$ yarn build
yarn run v1.22.4
$ node scripts/build.js
$ /home/nmummau/Code/vuetify/node_modules/.bin/lerna run build --stream
lerna notice cli v3.16.2
lerna info Executing command in 3 packages: "yarn run build"
vuetify: $ concurrently "yarn build:dist --no-progress" "yarn build:lib" -n "dist,lib" --kill-others-on-fail -r && yarn build:json
vuetify: /bin/sh: 1: concurrently: not found
vuetify: error Command failed with exit code 127.
vuetify: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run build exited 127 in 'vuetify'
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Done in 0.72s.
```

### Expected Behavior
Setting up local environment works.

### Actual Behavior
Unable to run `yarn build` with success. 

### Reproduction Link
<a href="https://vuetifyjs.com/en/getting-started/contributing/#setup-dev-environment" target="_blank">https://vuetifyjs.com/en/getting-started/contributing/#setup-dev-environment</a>

**Comment by KaelWD** [comment]
I think you need node 12

**Comment by nmummau** [comment]
> I think you need node 12

You were right, @KaelWD 
I downgraded to node v12.16.2 and now have my playground running now with `yarn dev`.

However, `yarn build` results in infinite loop of this error 
```
vuetifyjs.com: (node:5482) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 110)
```

Have you seen this before? Am I possibly also running another package that needs downgraded?

**Comment by MajesticPotatoe** [comment]
that warning is from `vuetifyjs.com` which is the docs, which is unrelated to `vuetify` playground. vuetify itself builds before the docs build (where this warning occurs). as for why your div isn't rendering, its hard to tell, whats your `playground.vue` file look like?

**Comment by nmummau** [comment]
Ah. I wasn't looking in `Playground.vue`. I started looking at `router.js` and assumed it would render `component1` defined in the router when on `localhost:8080/#/page1`. Poor assumption on my part. I've edited `Playground.vue` and now I am seeing my changes.

That all being said, this is definitely my user error. I'll be the first to admit. And I appreciate you guys helping me out. I hope this issue helps others too who might run into the same issues.
